Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/4E0O-Jiva-Volume-scaleup/jiva-volume-scaleup

```diff
@@ -0,0 +1,266 @@
+ | appNamespace      | env    | app-busybox-ns                      | jiva-replica-scaleup                             | 
+ | Litmus job label  | label  | liveness: litmus-busybox-liveness   | liveness: busybox-liveness-app-rep-scaleup       |
+ | appLabel          | env    | app=busybox-sts                     | app=busybox-jiva-replica                         | 
+  ----------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/value: app-busybox-ns/value: jiva-replica-scaleup/g' \
+-e 's/app=busybox-sts/app=bb-app-kill-jiva/g' \
+-e 's/liveness: litmus-busybox-liveness/liveness: busybox-liveness-app-rep-scaleup/g' busybox_loadgen_jiva_rep_scaleup.yml
+
+cat busybox_loadgen_jiva_rep_scaleup.yml
+
+sed -i '/command:/i \
+          - name: RUN_ID\
+            value: '"$run_id"'\
+' busybox_loadgen_jiva_rep_scaleup.yml
+
+# Run the Litmus job and get the details of the litmus job from litmus_job_runner utils.
+bash ../Openshift-EE/utils/litmus_job_runner label='liveness:busybox-liveness-app-rep-scaleup' job=busybox_loadgen_jiva_rep_scaleup.yml
+cd ..
+# Get the cluster state Once the litmus jobs completed.
+bash Openshift-EE/utils/dump_cluster_state;
+# Update the e2e event for the job.
+bash Openshift-EE/utils/event_updater jobname:jiva-replica-scaleup $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id 4E0O 3-functional "Verify if the Jiva replicas can be scaled up" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+##Scaling up the Jiva replicas 
+
+#################
+# LitmusBook 3 #
+#################
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=scaleup-jiva-replica metadata="")
+echo $test_name
+
+cd litmus
+# copy the content of deployer run_litmus_test.yml into a different file to update the test specific parameters.
+cp experiments/functional/scale_jiva_replica/run_litmus_test.yml busybox_scale_jiva_rep.yml
+
+ # Update the environmental variables in litmus job spec.
+
+ : << EOF
+  ---------------------------------------------------------------------------------------------------------------------
+ | specAttribute     | kind   |         default value               | test specifc value                               |
+  ---------------------------------------------------------------------------------------------------------------------|
+ | appNamespace      | env    | app-percona-ns                      | jiva-replica-scaleup                             |  
+ | appPVC            | env    | percona-mysql-claim                 | busybox-jiva-replica-scaleup                     |
+  ----------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/value: app-percona-ns/value: jiva-replica-scaleup/g' \
+-e 
```

Comment:
@shashank855 remove this `-e` in next line use escape characters to perform sed operation